x-pack/plugins/infra/public/pages/metrics/hosts_landing/enable_host_view_page.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { ReactNode } from 'react';
+import { EuiPageTemplate, EuiImage, EuiSpacer } from '@elastic/eui';
+import { css } from '@emotion/react';
+import { useEuiBackgroundColor } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { useTrackPageview } from '@kbn/observability-plugin/public';
+import { MetricsPageTemplate } from '../page_template';
+import hostsLandingBeta from './hosts_landing_beta.svg';
+import { ExperimentalBadge } from '../hosts/components/experimental_badge';
+
+interface Props {
+  actions?: ReactNode;
+}
+
+export const EnableHostViewPage = ({ actions }: Props) => {
+  const backgroundColor = useEuiBackgroundColor('subdued');
+
+  useTrackPageview({ app: 'infra_metrics', path: 'hosts_feature_enable_landing_page' });
+  useTrackPageview({
+    app: 'infra_metrics',
+    path: 'hosts_feature_enable_landing_page',
+    delay: 15000,
+  });
```

Comment:
**question:** in my understanding, we want to track 2 times the pageview, the first time in the very beginning and again after 15 seconds, is it correct? Why do we want this?

Reply:
I checked other implementations of the `useTrackPageview` and everywhere we track it like that and I wanted to be also consistent with it. I think the idea is to track initially and also one more time when the content is loaded (so we are sure that the user stays on the page and sees the content).